apps/jsonrpc2/test/support/bridge_sync_mock.ex

```diff
@@ -97,6 +97,14 @@ defmodule JSONRPC2.BridgeSyncMock do
     GenServer.call(__MODULE__, {:get_uncle_count_by_block_number, block_number})
   end
 
+  def get_uncle_by_block_hash_and_index(block_hash, index) do
```

Comment:
Is there any sense in grouping these together as 
`get_uncle_block_by_index(block_hash, index) when is_binary(block_hash) do`
and
`get_uncle_block_by_index(block_number, index) when is_integer(block_number) do`
?

Reply:
@willmeister good idea. I'll do it in the next PR